packages/dev/core/src/FlowGraph/flowGraphDataConnection.ts

```diff
@@ -76,11 +69,21 @@ export class FlowGraphDataConnection<T> extends FlowGraphConnection<FlowGraphBlo
         }
     }
 
-    /**
-     * Verifies if the connection has had a value set or not, either through a connection or by
-     * setting it directly.
-     */
-    public get isSet(): boolean {
-        return this._isSet;
+    public getClassName(): string {
+        return "FGDataConnection";
+    }
+
+    public serialize(serializationObject: any = {}) {
+        super.serialize(serializationObject);
+        serializationObject.richType = {};
+        this.richType.serialize(serializationObject.richType);
+    }
+
+    public static Parse(serializationObject: any, ownerBlock: FlowGraphBlock): FlowGraphDataConnection<any> {
+        const obj = super.Parse(serializationObject, ownerBlock);

```

Comment:
super in static feels wrong. TBH i didn't even know it was working.. Please use the right object reference

Reply:
Will do, thanks